conbench/templates/run.html

```diff
@@ -73,7 +86,8 @@ <h4>Benchmark results in Run</h4>
               <a href="{{ url_for('app.batch', batch_id=benchmark.batch_id) }}">{{ benchmark.display_bmname }}</a>
             </td>
             <td>
-              <a href="{{ url_for('app.benchmark-result', benchmark_result_id=benchmark.id) }}">{{ benchmark.display_case_perm }}</a>
+              <a href="{{ url_for('app.benchmark-result', benchmark_result_id=benchmark.id) }}">{{
+              benchmark.display_case_perm }}</a>
```

Comment:
Everything here and below this line in this file was caused by my VS Code format-on-save feature, sorry. Our current linting system doesn't have a problem with these changes either before or after.